lib/question.js

```diff
@@ -1117,6 +1117,10 @@ module.exports = {
             urls.baseUrl = urlPrefix;
         }
 
+        if (variant.workspace_id) {
+          urls.workspaceUrl = `${urlPrefix}/workspace/${variant.workspace_id}`;
+        }
```

Comment:
```suggestion
        if (variant.workspace_id) {
          urls.workspaceUrl = `${urlPrefix}/workspace/${variant.workspace_id}`;
        } else {
          urls.workspaceUrl = null;
        }
```
@nwalters512 I think we still need to set the `null` case, unless you prefer leaving it `undefined`?

Reply:
Our code generally shouldn't be distinguishing between `null` and `undefined`, so I'm happy with the latter. If you feel strongly about making the `null` explicit, I won't stop you 🙂